.github/workflows/examples.yml

```diff
@@ -32,8 +32,12 @@ jobs:
 
     - name: Install WasmEdge
       run: |
-        VERSION=0.10.0
-        curl -sSf https://raw.githubusercontent.com/WasmEdge/WasmEdge/master/utils/install.sh | sudo bash -s -- -e all --version=$VERSION --tf-version=$VERSION --tf-deps-version=$VERSION --tf-tools-version=$VERSION --image-version=$VERSION --image-deps-version=$VERSION  -p /usr/local
+        VERSION=0.11.2
+        curl -sSf https://raw.githubusercontent.com/WasmEdge/WasmEdge/master/utils/install.sh | sudo bash -s -- -e all --version=$VERSION --tf-version=$VERSION --tf-deps-version=$VERSION --tf-tools-version=$VERSION --image-version=$VERSION --image-deps-version=$VERSION -p /usr/local
+        curl -sLO https://github.com/WasmEdge/WasmEdge/releases/download/0.11.2/WasmEdge-plugin-wasi_crypto-0.11.2-manylinux2014_x86_64.tar.gz
```

Comment:
this workflows is run on ubuntu-20.04, so we should use WasmEdge-plugin-wasi_crypto-0.11.2-[ubuntu20.04_x86_64.tar.gz](https://github.com/WasmEdge/WasmEdge/releases/download/0.11.2/WasmEdge-plugin-wasi_crypto-0.11.2-ubuntu20.04_x86_64.tar.gz)

Reply:
It seems like only wasmedge-tensorflow-lite didn't recognize the wasi_crypto plugin.

Reply:
wasmedge-tensorflow-lite may have some bug. please add a feature that is about crypto, and close this feature in tensorflow test.